### PR TITLE
Add dependent refresh zoom

### PR DESCRIPTION
**Pull request by vadimbakaev**

### CODE REVIEW

File: src/components/world-map.vue [hunk 1]
```diff
@@ -109,10 +117,13 @@ export default class extends Vue {
...
     } = this.map.getBoundingBox();
 
+    const zoom = this.map.getZoom();
+    const updateRate = this.fibonacci(Math.max(0, 14 - zoom));
+
     return {
       '@type': types.startFlightList,
       'maxFlights': store.getters.maxFlights,
-      'updateRate': store.getters.updateRate,
+      'updateRate': zoom,
```
Comment (simoexpo) [inline src/components/world-map.vue]: `zoom` or `updateRate`?

**Comment by simoexpo** [comment]
With this logic we set 1s as the min updateRate right? Are we ok with this? It's not a bit too low?

File: src/components/world-map.vue (hunk 1 above)
Comment (vadimbakaev) [reply src/components/world-map.vue]: Fixed

Comment (vadimbakaev) [reply src/components/world-map.vue]: updateRate is correct

**Comment by lodamar** [comment]
> With this logic we set 1s as the min updateRate right? Are we ok with this? It's not a bit too low?

I agree with Simone, refresh rate should be between 5 and 30 (maybe 60) seconds

**Comment by vadimbakaev** [comment]
> > With this logic we set 1s as the min updateRate right? Are we ok with this? It's not a bit too low?
> 
> I agree with Simone, refresh rate should be between 5 and 30 (maybe 60) seconds

Added update limit between 30 and 144.

**Pull request merged**